app/controllers/renewals_controller.rb

```diff
@@ -55,38 +62,33 @@ def deny_access
     def bulk_renewal_flash(response, type)
       return unless response[type].any?
 
-      flash[type] = I18n.t(
-        "renew_all_items.#{type}_html",
-        count: response[type].length,
-        items: content_tag(
-          :ul,
-          safe_join(response[type].map do |renewal|
-            content_tag(:li,
-                        renewal_prompt(renewal),
-                        {},
-                        false)
-          end, '')
-        )
-      )
+      items = response[type].map do |renewal|
```

Comment:
It's even less lines of code! :) Thanks, to me this is so much clearer.

Reply:
I agree! thanks!